rfcs/text/0044-add-apple-platform-specific-fields.md

```diff
@@ -0,0 +1,119 @@
+# 0000: Name of RFC
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **0 (strawperson)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **TBD** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+
+### Summary
+This RFC proposes the addition of Apple platform-specific fields to the ECS schema. This enhancement will enable security software vendors to more accurately map out data, particularly for Apple platforms.
+
+The following feelds needs to be considered being added:
+
+## Fields
+
+##### Proposed New Fields for Process object
+
+Field | Type | Example | Description
+--- | --- | --- | ---
+process.responsible	| keyword	| Terminal.app	| The responsible process on macOS, from an ancestry perspective, is the process that originally launched or spawned a given process.
```

Comment:
will it be then `process.process.responsible` as a full name including namespace?

Reply:
Hmm no should be process.responsible.* @trisch-me

Reply:
@trisch-me I updated the .dm file, but can you double check the `process.yml`, it's my first time making suggestions for ECS so i'm new here!